ansible_collections/arista/avd/roles/eos_designs/doc/l3ls-evpn/fabric-topology.md

```diff
@@ -422,8 +422,40 @@ l2leaf:
           id: < integer >
           mgmt_ip: < IPv4_address/Mask >
           l3leaf_interfaces: [ < ethernet_interface_8 >, < ethernet_interface_8 > ]
+
+    # node_group_3, will result in Active/Active connection to L3LEAFs.
+    < node_group_3 >:
+      parent_l3leafs: [ DC1-SVC3A ]
+      mlag: false
+      nodes:
+
+        # First node.
+        < l2_leaf_inventory_hostname_2 >:
+          id: < integer >
+          mgmt_ip: < IPv4_address/Mask >
+          l3leaf_interfaces: [ < ethernet_interface_7 >, < ethernet_interface_7 > ]
+          short_esi: < short esi value >
```

Comment:
I would think this should be under the node_group and not the individual node. If we have l2leafs in MLAG, they would share the same ESI on the upstream l3leafs.

Reply:
Moved at node_group level with commit e3c7310